paasta_tools/cli/cmds/secret.py

```diff
@@ -423,7 +423,8 @@ def paasta_secret(args):
                 get_secret(
                     kube_client,
                     get_paasta_secret_name(namespace, service, args.secret_name),
-                    namespace,
+                    key_name=args.secret_name,
+                    namespace=namespace,
```

Comment:
@nurdann Can you take a look at the changes in this file and make sure they make sense? I was able to reproduce this bug locally by setting `secrets_for_owner_team: true` in my local yelpsoa-configs' `paasta-contract-monitor/service.yaml` and then running `paasta secret decrypt -s paasta-contract-monitor -n goss -c norcal-devc -y ~/pg/yelpsoa-configs/`.  This fails with 

```
Traceback (most recent call last):
  File "/opt/venvs/paasta-tools/bin/paasta", line 11, in <module>
    sys.exit(main())
  File "/opt/venvs/paasta-tools/lib/python3.8/site-packages/paasta_tools/cli/cli.py", line 256, in main
    return_code = args.command(args)
  File "/opt/venvs/paasta-tools/lib/python3.8/site-packages/paasta_tools/cli/cmds/secret.py", line 432, in paasta_secret
    get_secret(
  File "/opt/venvs/paasta-tools/lib/python3.8/site-packages/paasta_tools/kubernetes_tools.py", line 4157, in get_secret
    secret_data = kube_client.core.read_namespaced_secret(
KeyError: 'paasta'
```

By running `/nail/home/krall/pg/paasta/.tox/py38-linux/bin/paasta secret decrypt -s paasta-contract-monitor -n goss -c norcal-devc -y ~/pg/yelpsoa-configs/` (i.e. using this branch's version of secret.py), I get the value of the secret as desired.